### PR TITLE
Added support for python caldav package

### PR DESCRIPTION
**Pull request by marenz2569**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/caldav/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  propagatedBuildInputs = [ tzlocal requests vobject lxml ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ll9knpc50yxx858hrvfnapdi2a6g1pz9cnjhwffry2x7r4ckarz";
+  };
+
+  meta = with lib; {
+    description = "This project is a CalDAV (RFC4791) client library for Python.";
+    homepage = "http://pythonhosted.org/caldav/";
```
Comment (Infinisil) [inline pkgs/development/python-modules/caldav/default.nix]: Use the HTTPS version

File: pkgs/development/python-modules/caldav/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    license = licenses.apache2;
```
Comment (Infinisil) [inline pkgs/development/python-modules/caldav/default.nix]: The license attribute is `asl20`, not `apache2`

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -2921,6 +2921,11 @@
     github = "marcweber";
     name = "Marc Weber";
   };
+	marenz = {
+		email = "marenz@arkom.men";
+		github = "marenz2569";
+		name = "Markus Schmidl";
+	};
```
Comment (Infinisil) [inline maintainers/maintainer-list.nix]: Make this a separate commit in this PR

**Review by infinisil — CHANGES_REQUESTED** [review]
Also change the commit message for adding the package to "pythonPackages.caldav: init at 0.6.0" as is convention

File: pkgs/development/python-modules/caldav/default.nix (hunk 1 above)
Comment (teto) [reply pkgs/development/python-modules/caldav/default.nix]: you can remove quotes as well

**Comment by marenz2569** [comment]
@Infinisil I made all the requested changes.

**Review by infinisil — APPROVED** [review]

**Comment by Infinisil** [comment]
@GrahamcOfBorg build python2.pkgs.caldav
@GrahamcOfBorg build python3.pkgs.caldav
